build.gradle

```diff
@@ -21,9 +21,4 @@ allprojects {
         jcenter()
         google()
     }
-}
-
-task clean(type: Delete) {
-    delete rootProject.buildDir
-    delete 'CHANGELOG.md.release'
-}
```

Comment:
why is this no longer needed?

Reply:
This is not included as part of the OSS gradle plugin, see https://github.com/auth0/oss-library-gradle-plugin/blame/429ca29ecc1439117aa0f858bfd3c462277c3543/src/main/groovy/com/auth0/gradle/oss/RootProjectPlugin.groovy#L19-L21. The `rootProject.buildDir` is now deleted by the base android gradle plugin. We just extend that and delete the changelog if present.